src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -247,6 +248,21 @@ public string FileName
             && !string.IsNullOrWhiteSpace(this.Message)
             && this.Message != this.ShortMessage;
 
+        [Browsable(false)]
+        public string XamlMessage
+        {
+            get
+            {
+                string value = null;
+                if (this.SarifResult?.Message?.TryGetProperty(XamlPropertyName, out value) == true)
```

Comment:
>XamlPropertyName [](http://example.com/codeflow?start=62&length=16)

Is the value of this ever going to change after the error list item is created? If not, then we can cache the value on first read, unless there is a space complexity concern.